server/s3_put_multiobjectcopy_action.h

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ *
+ */
+
+#pragma once
+
+#ifndef __S3_SERVER_S3_PUT_MULTIOBJECTCOPY_ACTION_H__
```

Comment:
we might have to follow the same but cortx-s3 nomenclature seems a little lengthy, S3_PUT_PARTCOPY instead of S3_PUT_MULTIOBJECTCOPY seems better to me as the concept of part is there only if the request is multipart.